index.html

```diff
@@ -78,6 +83,231 @@ <h3 class="connect">LET'S CONNECT</h3>
+        </div>
+      </div>
+      <div class="second-page">
+        <img
+          src="./images/mobile-images/Avail.png"
+          width="295"
+          height="220"
+          alt="image"
+          class="pics"
+        />
+        <div class="img-content">
+          <h3 class="project-title">Multi-Post <br />Stories</h3>
+          <ul class="sub-title">
+            <li class="sub-title-topic">CANOPY</li>
+            <li class="sub-title-other-words">Back End Dev</li>
+            <li class="sub-title-other-words">2015</li>
+          </ul>
+          <p class="project-content">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="tech">
+            <li class="tech-type">HTML</li>
+            <li class="tech-type">CSS</li>
+            <li class="tech-type">JavaScript</li>
+          </ul>
+          <button class="project-button">See project</button>
+        </div>
+      </div>
+      <div class="second-page">
+        <img
+          src="./images/mobile-images/gym.png"
+          width="295"
+          height="220"
+          alt="image"
+          class="pics"
+        />
+        <div class="img-content">
+          <h3 class="project-title">Tonic</h3>
+          <ul class="sub-title">
+            <li class="sub-title-topic">CANOPY</li>
+            <li class="sub-title-other-words">Back End Dev</li>
+            <li class="sub-title-other-words">2015</li>
+          </ul>
+          <p class="project-content">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="tech">
+            <li class="tech-type">HTML</li>
+            <li class="tech-type">CSS</li>
+            <li class="tech-type">JavaScript</li>
+          </ul>
+          <button class="project-button">See project</button>
+        </div>
+      </div>
+      <div class="second-page">
+        <img
+          src="./images/mobile-images/art.png"
+          width="295"
+          height="220"
+          alt="image"
+          class="pics"
+        />
+        <div class="img-content">
+          <h3 class="project-title">Multi-Post <br />Stories</h3>
+          <ul class="sub-title">
+            <li class="sub-title-topic">CANOPY</li>
+            <li class="sub-title-other-words">Back End Dev</li>
+            <li class="sub-title-other-words">2015</li>
+          </ul>
+          <p class="project-content">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="tech">
+            <li class="tech-type">HTML</li>
+            <li class="tech-type">CSS</li>
+            <li class="tech-type">JavaScript</li>
+          </ul>
+          <button class="project-button">See project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about-page">
```

Comment:
The top right part of this section is not matching the Figma template of your choosing. There is a certain border-radius for the top right part defined in the provided design which is missing in your case. Kindly address this issue before submitting for a re-review.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://github.com/Kwamay/My-Porforlio/assets/33268927/ad16df1a-6dbd-4e81-b9e6-6c2f91c0c222) | ![image](https://github.com/Kwamay/My-Porforlio/assets/33268927/4302c2f6-c642-4757-83a2-497ab75ce7e0) |